lib/view/templates.py

```diff
@@ -220,25 +220,6 @@ def usage_weighted_avg(edatas: list[EntryData]):
         namespace_field,
         node_field,
         hidden_node_id_field,
-        Field(
```

Comment:
removed because it is also in `info namespace object` where it probably fits better